### PR TITLE
DOCSP-13035: CSFLE Azure and Java

### PR DESCRIPTION
**Pull request by ccho-mongodb**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-13035
Note: this addresses Java driver changes for Azure support only. All the other combinations will be in separate tickets.

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/b96ed43/drivers/docsworker-xlarge/DOCSP-13035-csfle-azure-java/security/client-side-field-level-encryption-local-key-to-kms

For relevant changes, select the Azure Key Vault tab and Java driver tab within that tabset.

### CODE REVIEW

File: source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml [hunk 1]
```diff
@@ -169,29 +178,25 @@ content: |
      .. tab::
         :tabid: java-sync
 
-        .. code-block:: Java
+        In your call to ``createDataKey()`` on a ``ClientEncryption`` keyvault
+        instance in ``CSFLEHelpers.java``, add your KMS provider and master 
+        key details as follows:
```
Comment (terakilobyte) [inline source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml:183]: Small suggestion to make it less wordy.
```suggestion
        In your ``ClientEncryption.createDataKey()`` call in ``CSFLEHelpers.java``, add your KMS provider 
        and master key details as follows:
```

**Review by terakilobyte — APPROVED** [review]
Copy LGTM, left an optional suggestion.

**Review by kanchana-mongodb — APPROVED** [review]
LGTM

File: source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml (hunk 1 above)
Comment (ccho-mongodb) [reply source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml:183]: To me, this implies that `createDataKey()` is a static method, but it's an instance method. Maybe if I switch around the order of the sentence to:
"In `CSFLEHelpers.java`, add your KMS provider and master key details to your call to `createDataKey()` on your `ClientEncryption` instance."

Which loses some specificity into what ClientEncryption represents, but maybe that info is not that important.

**Pull request merged**